config/projects.yaml

```diff
@@ -247,8 +247,14 @@ doctrine-orm-admin-bundle:
             php: ['7.3', '7.4', '8.0']
             php_extensions: ['zip']
             variants:
-                symfony/symfony: ['4.4']
+                symfony/symfony: ['4.4', '5.3']
                 sonata-project/admin-bundle: ['dev-master']
+        4.x:
+            php: ['7.3', '7.4', '8.0']
+            php_extensions: ['zip']
+            variants:
+                symfony/symfony: ['4.4', '5.3']
+                sonata-project/admin-bundle: ['^4@rc']
```

Comment:
why don't you use directly 4?

Reply:
Indeed the current check is 
```
'dev-master' === $this->version ? $this->version : ($this->version.'.*')
```
so it won't work like this.

Reply:
But 4.* doesn't work https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1488